app/services/invoices/calculate_fees_service.rb

```diff
@@ -296,6 +296,8 @@ def termination_boundaries(subscription, boundaries)
       current_time = Time.zone.at(timestamp)
       current_time_in_timezone = Time.zone.at(timestamp).in_time_zone(customer.applicable_timezone)
 
+      return boundaries if (current_time_in_timezone - 1.day) < subscription.started_at
```

Comment:
can we add a comment to explain it here? 
not sure to understand next time I'll see this line 😅